### PR TITLE
[Chore] Test timeout for non-multitenant TempoStack and TempoMonolithic instances.

### PR DESCRIPTION
**Pull request by IshwarKanse**
tests/e2e-openshift/route -- Test with custom timeout set to 2m
tests/e2e-openshift/monolithic-route -- Test with custom timeout set to 2m
tests/e2e-openshift/tls-singletenant -- Test default timeout of 30s
tests/e2e-openshift/tls-monolithic-singletenant/ -- Test default timeout of 30s

```
--- PASS: chainsaw (0.00s)
    --- PASS: chainsaw/monolithic-route (55.16s)
    --- PASS: chainsaw/tls-singletenant-monolithic (96.14s)
    --- PASS: chainsaw/route (138.22s)
    --- PASS: chainsaw/tls-singletenant (164.25s)
PASS
Tests Summary...
- Passed  tests 4
- Failed  tests 0
- Skipped tests 0
Done.
```
Once the multitenancy issues are fixed, will add the asserts for the multitenancy tests as well.

### CODE REVIEW

**Comment by IshwarKanse** [comment]
cc @pavolloffay

**Comment by IshwarKanse** [comment]
Found an issue with Tempo configmap not being serialised in readable format when distributor tls is enabled in TempoStack instance. https://issues.redhat.com/browse/TRACING-4833

File: tests/e2e-openshift/tls-singletenant/01-assert.yaml [hunk 1]
```diff
@@ -104,15 +116,133 @@ spec:
...
+  tls:
+    termination: reencrypt
+  to:
+    kind: Service
+    name: tempo-simplest-query-frontend
+    weight: 100
+  wildcardPolicy: None
+
+---
+apiVersion: v1
```
Comment (pavolloffay) [inline tests/e2e-openshift/tls-singletenant/01-assert.yaml:152]: Can we format this file?

Comment (IshwarKanse) [reply tests/e2e-openshift/tls-singletenant/01-assert.yaml:152]: Its a bug as mentioned in the above comment. https://issues.redhat.com/browse/TRACING-4833 without the distributor tls enabled, the formatting is serialised correctly.

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**